### PR TITLE
fix: try to do calculations on numbers and strings multiple choice questions

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

This updates and reorders the multiple choice questions for this lecture based on the current video lecture.

### CODE REVIEW

**Review by huyenltnguyen — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/lecture-working-with-numbers-booleans-and-the-math-object/67327195e77b1bd90bdd49d7.md [hunk 1]
```diff
@@ -14,107 +14,115 @@ Watch the lecture video and answer the questions below.
...
 
 ### --feedback--
 
-Consider how JavaScript interprets `true` and `false` in numerical contexts.
+Recall how type coercion works for addition operations, and how you use the same symbol to combine strings.
 
 ---
 
-JavaScript treats `true` as `1` and `false` as `0` in arithmetic operations.
+JavaScript converts the number `3` into the string `”3”`, concatenates the two strings, and assigns the value `”319”` to result.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lecture-working-with-numbers-booleans-and-the-math-object/67327195e77b1bd90bdd49d7.md:41]:
```suggestion
JavaScript converts the number `3` into the string `"3"`, concatenates the two strings, and assigns the value `"319"` to result.
```

File: curriculum/challenges/english/25-front-end-development/lecture-working-with-numbers-booleans-and-the-math-object/67327195e77b1bd90bdd49d7.md [hunk 2]
```diff
@@ -14,107 +14,115 @@ Watch the lecture video and answer the questions below.
...
 
 ---
 
-JavaScript converts the boolean to a string before performing the operation.
+JavaScript converts the string `“19”` into the number `19`, performs the operation, and assigns the value `22` to result.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lecture-working-with-numbers-booleans-and-the-math-object/67327195e77b1bd90bdd49d7.md:45]:
```suggestion
JavaScript converts the string `"19"` into the number `19`, performs the operation, and assigns the value `22` to result.
```

File: curriculum/challenges/english/25-front-end-development/lecture-working-with-numbers-booleans-and-the-math-object/67327195e77b1bd90bdd49d7.md [hunk 3]
```diff
@@ -14,107 +14,115 @@ Watch the lecture video and answer the questions below.
...
 
 ### --feedback--
 
-Think about how JavaScript handles different data types in math operations.
+Recall how type coercion works for subtraction, multiplication, division, and other kinds of operations.
 
 ---
 
-JavaScript throws an error when you try to mix strings and numbers in arithmetic.
+JavaScript converts the number `4` into the string `”4”`, concatenates the two strings, and assigns the value `”64”` to result.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lecture-working-with-numbers-booleans-and-the-math-object/67327195e77b1bd90bdd49d7.md:73]:
```suggestion
JavaScript converts the number `4` into the string `"4"`, concatenates the two strings, and assigns the value `"64"` to result.
```

File: curriculum/challenges/english/25-front-end-development/lecture-working-with-numbers-booleans-and-the-math-object/67327195e77b1bd90bdd49d7.md [hunk 4]
```diff
@@ -14,107 +14,115 @@ Watch the lecture video and answer the questions below.
...
+### --feedback--
 
-JavaScript converts the number into a string before performing the operation.
+Recall how type coercion works for subtraction, multiplication, division, and other kinds of operations.
 
-### --feedback--
+---
 
-Think about how JavaScript handles different data types in math operations.
+JavaScript converts the string `“6”` into the number `6`, performs the operation, and assigns the value `2` to `result`.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lecture-working-with-numbers-booleans-and-the-math-object/67327195e77b1bd90bdd49d7.md:89]:
```suggestion
JavaScript converts the string `"6"` into the number `6`, performs the operation, and assigns the value `2` to `result`.
```

**Comment by scissorsneedfoodtoo** [comment]
Thanks for catching all these issues with special characters. Seems like it might be a product of copying from the HN editor into VS Code.

**Review by Sembauke — APPROVED** [review]

**Pull request merged**